src/test/scala/mesosphere/UnitTest.scala

```diff
@@ -79,26 +79,51 @@ trait ValidationTestLike extends Validation {
     case th => throw th
   }.get
 
-  def containViolation(tuple: (String, String)): Matcher[Result] = containViolation(tuple._1, tuple._2)
+  private def describeViolation(c: ConstraintViolation) =
+    s"""- "${c.path}" -> "${c.constraint}""""
 
-  def containViolation(path: String, message: String): Matcher[Result] = {
-    Matcher {
-      case Success =>
-        MatchResult(
-          false,
-          s"result had no violations; expected ${path} -> ${message}",
-          s"result was success")
+  def shouldViolate[T](entity: T, expectedViolations: (String, String)*)(implicit validator: Validator[T]): Unit = {
```

Comment:
Why make the validator implicit?

Reply:
Great question; I actually sure about this change, but went with it since it it was what MV had proposed in his branch. I really disliked the wix accord matchers. I find the error messages to be cryptic and the matchers to be particularly verbose to write. Things are better with wix 0.7, but I still prefer the simple "haveViolations" matcher.

I'll rewrite this to exclude the `shouldFail` and `shouldSucceed` with our opinionated matchers (path and violation message only matter). should `haveViolations` and `haveNoViolations`